Dockerfile-slim.template

```diff
@@ -0,0 +1,33 @@
+FROM buildpack-deps:jessie-curl
```

Comment:
The 0.10 and 0.12 images historically used debian:jessie for "slim" and then add the curl ca-certificates xz-utils packages via apt-get. 

I think standardizing on buildpack-deps:jessie-curl (and just installing xz-utils) is probably the better way to go and we should do that in a separate PR for the 0.10 and 0.12 slim variants.